modules/ROOT/pages/database-administration/standard-databases/listing-databases.adoc

```diff
@@ -171,6 +181,8 @@ The value is a string formatted as:
 ----
 A database must be `online` or `deallocating` for this value to be available.
 For other database states the value will be `NULL`.
+
+The value will be `NULL` for composite databases, as it is not applicable for them.
```

Comment:
> Where is this updated store column (that doesn't say --0.0 in 2025.04)? We need to mark the change with a label that it's available from 2025.04, but I don't see the change.

This is the store column, it's really just a bug fix as `--0.0` isn't really a valid/useful storage engine/format. But we're not backporting it to the other calendar releases just the 5.LTS one 🤷

Reply:
both the `store`, `currentPrimariesCount` and `currentSecondariesCount` columns got updated to return `null` for composite (the current_Count ones are also columns that now have this sentence)

Reply:
Maybe we should add a footnote to explain that this change is applicable to 2025.04 and 5.25.5 and later.

Reply:
5.26.5 maybe? but sure could work (but I don't know how to do a footnote)